capture_gui/app.py

```diff
@@ -556,8 +556,9 @@ def _read_widget_configuration(self):
         """Read the stored widget inputs"""
         inputs = {}
 
-        if not os.path.isfile(self.settingfile) or \
-                        os.stat(self.settingfile).st_size == 0:
+        is_file = os.path.isfile(self.settingfile)
+        file_size = os.stat(self.settingfile).st_size
```

Comment:
As a note:
Looking over the code it currently already assures the config file exists because it calls `_ensure_config_exist()` near the start of the the `__init__` prior to this function call. Yet that might also make it arguable whether we need to check for `os.path.isfile()`.